src/components/QrReader/qrReader.js

```diff
@@ -1,11 +1,8 @@
 import React from "react";
-import dynamic from "next/dynamic";
 import PropTypes from "prop-types";
+import QrReader from "react-qr-reader";
 import { getLogger } from "../../utils/logger";
 
-// Need to dynamically import react-qr-reader as it access window directly
```

Comment:
not sure about the implication of this comment.. @yehjxraymond can you look into it. Seems to work fine without loading dynamically.

Reply:
SSR, we dont care without next-js